main/linux.py

```diff
@@ -46,7 +48,7 @@ def convert_linux(filepath: str, files, converted_file_path: str, has_type_in_re
 
     command = f"cd {filepath}"
     for type_in in has_type_in_request:
-        if has_type_in_request[type_in] and type_in is not ".pdf":
+        if has_type_in_request[type_in] and type_in != ".pdf":
```

Comment:
`is` checks for object identity, whereas `!=` checks for value identity. Here, probably, you should check for value identity.

Suggestion: rollback changes

Reply:
My mistake. It is ok.